Core/include/Acts/Seeding/Seedfinder.ipp

```diff
@@ -364,12 +368,14 @@ void Seedfinder<external_spacepoint_t, platform_t>::createSeedsForGroup(
         // from rad to deltaCotTheta
         float p2scatterSigma = pT2scatterSigma * iSinTheta2;
         // if deltaTheta larger than allowed scattering for calculated pT, skip
-        if (m_config.enableCutsForSortedSP) {
+        if (m_config.cotThetaSorting) {
```

Comment:
This is indeed a cut since it changes the number of seeds. At first, I wrote in the code that 
_"since cotThetaB > cotThetaT and the SP are sorted by cotTheta, the next bottom SP is expected to have cotThetaB > cotThetaT as well and deltaCotTheta2 - error2 > sigmaSquaredScatteringMinPt"_
but we realised that this is not always true since error2 is not constant. 

This cut was probably optimised for ITk pixel, and it is not used for strips.

I will change the variable names again in a way that I hope will make more sense:

`enableCutsForSortedSP` to enable the sorting and `skipPreviousTopSP` for the additional cut to skip top SPs if cotThetaBB < cotThetaT

@noemina @robertlangenberg

Reply:
For the ITk strips there is no cut on the scattering? None on the error? That is very surprising. Really, none?

Reply:
There is the cut on the scattering that accounts for the contribution to the minimum pt and also the one on
the actual estimated pT, what I mean is that there is no additional skip top SPs if cotThetaBB < cotThetaT

Reply:
ok my actual qualms with this wasn't the added if-statement, but the changed logic for the combination of deltaCotTheta, scattering and error.

Reply:
I have since understood the error of my ways. The original error calculation code is wrong, so please don't hide it behind the enableCutsForSortedSP. Just remove the "else" part and don't make it conditional on this flag. If I understood correctly, this means the enableCutsForSortedSP flag is not necessary anymore.

Reply:
Yes, you are right. I also think the code should be documented in order to explain why this is the correct calculation and avoid re-turning on this in the future.

Reply:
ok! I will do that and also document the code